src/typescript/services/apiService.ts

```diff
@@ -0,0 +1,24 @@
+class ApiService {
+    private baseUrl: string;
+    private path: string;
+
+    constructor(baseUrl: string, path: string) {
+        this.baseUrl = baseUrl;
+        this.path = path;
+    }
+
+    async get(): Promise<any> {
+        try {
+            const response = await fetch(`${this.baseUrl}${this.path}?_expand=category`);
+            if (!response.ok) {
+                throw new Error('Network response was not ok');
```

Comment:
https://dev.to/dionarodrigues/fetch-api-do-you-really-know-how-to-handle-errors-2gj0?ref=dailydev

Read article pls, I don't think this is network issue

Reply:
I fixed @phuocduong-agilityio . Please help me check. Thanks!